sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationMockTests.cs

```diff
@@ -776,6 +776,72 @@ public void AnalyzeOperationExtractSummaryWithTwoActions()
 
         #endregion Extract summary
 
+        #region Multi Category Classify
+
+        [Test]
+        public async Task AnalyzeOperationMultiCategoryClassifyWithDisableServiceLogs()
+        {
+            var mockResponse = new MockResponse(202);
+            mockResponse.AddHeader(new HttpHeader("Operation-Location", "something/jobs/2a96a91f-7edf-4931-a880-3fdee1d56f15"));
+
+            var mockTransport = new MockTransport(new[] { mockResponse, mockResponse });
+            var client = CreateTestClient(mockTransport);
+
+            var documents = new List<string>
+            {
+                "Elon Musk is the CEO of SpaceX and Tesla."
+            };
+
+            var actions = new MultiCategoryClassifyAction(FakeProjectName, FakeDeploymentName)
+            {
+                DisableServiceLogs = true
+            };
+
+            TextAnalyticsActions batchActions = new TextAnalyticsActions()
+            {
+                MultiCategoryClassifyActions = new List<MultiCategoryClassifyAction>() { actions },
+            };
+
+            await client.StartAnalyzeActionsAsync(documents, batchActions);
+
+            var contentString = GetString(mockTransport.Requests.Single().Content);
+            string logging = contentString.Substring(contentString.IndexOf("loggingOptOut"), 19);
+
+            var expectedContent = "loggingOptOut\":true";
+            Assert.AreEqual(expectedContent, logging);
+        }
+
+        [Test]
+        public void AnalyzeOperationMultiCategoryClassifyWithTwoActions()
+        {
+            var mockResponse = new MockResponse(202);
+            mockResponse.AddHeader(new HttpHeader("Operation-Location", "something/jobs/2a96a91f-7edf-4931-a880-3fdee1d56f15"));
+
+            var mockTransport = new MockTransport(new[] { mockResponse, mockResponse });
+            var client = CreateTestClient(mockTransport);
+
+            var documents = new List<string>
+            {
+                "Elon Musk is the CEO of SpaceX and Tesla."
+            };
+
+            TextAnalyticsActions batchActions = new()
+            {
+                MultiCategoryClassifyActions = new List<MultiCategoryClassifyAction>()
+                {
+                    new MultiCategoryClassifyAction(FakeProjectName, FakeDeploymentName),
+                    new MultiCategoryClassifyAction(FakeProjectName, FakeDeploymentName)
+                    {
+                        DisableServiceLogs = true
+                    }
+                },
+            };
+
+            ArgumentException ex = Assert.ThrowsAsync<ArgumentException>(async () => await client.StartAnalyzeActionsAsync(documents, batchActions));
+            Assert.AreEqual("Multiple of the same action is not currently supported.", ex.Message);
```

Comment:
I believe this test should work if different values for project name / deployment name are passed for each action

Reply:
not following the logic we currently have in the .NET SDK.
I prefer to leave this as is so we can make progress in this PR, work on the custom entities PR, and once those 2 are merged, then focus on the work on https://github.com/Azure/azure-sdk-for-net/issues/24430